main.tf

```diff
@@ -44,7 +44,7 @@ locals {
   private_key_rsa_bits             = "4096"
   release_prefix                   = "release"
   resource_name                    = "${local.name_prefix}-${local.build_id}"
-  scan_slug                        = "${var.s3_scan_bucket}/${local.wam_version}"
+  scan_slug                        = "${var.s3_scan_bucket}/${local.wam_version}/scans"
```

Comment:
Where does this path end up in practice? What does this actually end up doing?

Reply:
Here's an example of the current result:

https://watchmaker.dev.cloudarmor.io/list.html#releases/0.23.2.dev20210813120818/

Reply:
Another option is we can combine everything into a single variable to give the `scan_slug` more flexibility.

Reply:
I like combining them... Looks that that has minimal impact elsewhere, maybe `var.scan_s3_url` that takes the form `s3://<bucket>/<prefix>`?

Reply:
@lorengordon Ready for another look.  So I still kept the bucket and prefix as separate variables but this now allows more flexibility with the prefix.  It also minimizes the changes given how we're constructing the s3 url in the userdata scripts.

Reply:
Example with `releases/0.23.2.dev20210813120818/scap_scan_reports` prefix in `var.scan_s3_prefix`:

https://watchmaker.dev.cloudarmor.io/list.html#releases/0.23.2.dev20210813120818/scap_scan_reports/

Reply:
I see... It's still kinda ugly to construct this way... For linux, the `aws s3` command requires the `s3://` protocol specifier anyway. Windows is the annoying one, since the powershell commandlet does not support the same syntax. But I think we can just "trim-start" the `s3://` to make it work? Then it will be a little cleaner?

Reply:
For example:

```
> "s3://foo/bar".trimstart("s3://")
foo/bar
```

Reply:
Okay.  I'll try your suggestion.